splunk_add_on_ucc_framework/commands/build.py

```diff
@@ -508,6 +507,44 @@ def generate(
         logger.info(
             f"Installed add-on requirements into {ucc_lib_target} from {source}"
         )
+        if global_config.has_custom_search_commands():
```

Comment:
This condition can be removed, the for loop in the next line would handle  no custom search commands condition.

Reply:
Thanks for pointing out, removed the condition.